code/datums/mutations/body.dm

```diff
@@ -9,8 +9,8 @@
 	synchronizer_coeff = 1
 	power_coeff = 1
 
-/datum/mutation/human/epilepsy/on_life()
-	if(prob(1 * GET_MUTATION_SYNCHRONIZER(src)) && owner.stat == CONSCIOUS)
+/datum/mutation/human/epilepsy/on_life(delta_time, times_fired)
+	if(DT_PROB(0.5 * GET_MUTATION_SYNCHRONIZER(src), delta_time) && owner.stat == CONSCIOUS)
```

Comment:
I'm just gonna go ahead and assume all of these sorts of changes are sane, otherwise I'm gonna go insane.  Is that a reasonable assumption to make?

Reply:
Yes. For both.